Examples/SearchExamples/SimpleUISearchViewController.swift

```diff
@@ -19,6 +19,25 @@ class SimpleUISearchViewController: MapsViewController {
         let cameraOptions = CameraOptions(center: .sanFrancisco, zoom: 15)
         mapView.camera.fly(to: cameraOptions, duration: 1, completion: nil)
 
+        let horizontalMargin: CGFloat = 4
+        mapView.ornaments.options.logo = LogoViewOptions(
+            position: .topLeading,
+            margins: CGPoint(x: horizontalMargin, y: 0)
+        )
+
+        mapView.ornaments.options.attributionButton = AttributionButtonOptions(
+            position: .topLeading,
+            margins: CGPoint(x: 0, y: 4)
+        )
+
+        let scalarX = mapView.ornaments.logoView.bounds.maxX + 12
+        mapView.ornaments.options.scaleBar = ScaleBarViewOptions(
+            position: .topLeading,
+            margins: CGPoint(x: scalarX, y: 0),
+            visibility: .visible,
+            useMetricUnits: false
+        )
```

Comment:
I think we can just hide scale bar, not like it's important to the sample

Reply:
Yes it is not essential although I would like to leave it because the `margin` behavior is not entirely obvious so this helps demonstrate what is possible.